lib/Conversion/TorchToTcp/TcpCustomOp.cpp

```diff
@@ -146,6 +147,89 @@ class ConvertAten_IndexPutImplOp
   }
 };
 
+class ConvertAtenConvolutionOp : public OpConversionPattern<AtenConvolutionOp> {
+public:
+  using OpConversionPattern<AtenConvolutionOp>::OpConversionPattern;
+
+  LogicalResult
+  matchAndRewrite(AtenConvolutionOp op, OpAdaptor adaptor,
+                  ConversionPatternRewriter &rewriter) const override {
+    SmallVector<Type> resultTypes;
+    if (failed(OpConversionPattern<AtenConvolutionOp>::getTypeConverter()
+                   ->convertTypes(op->getResultTypes(), resultTypes))) {
+      return failure();
+    }
+
+    SmallVector<Value> operands;
+    SmallVector<StringRef> operandNames;
```

Comment:
Naming nit: Prefer `tensorOperandNames` (to distinguish from non-tensor operands that we convert to attributes).

Reply:
actually, never-mind since we seem to use operands to refer to tensorOperands always so there is no ambiguity..